service/src/main/java/aero/minova/cas/ldap/MultipleLdapServerAddressesUserDetailsManager.java

```diff
@@ -10,6 +10,8 @@
 import org.springframework.security.ldap.userdetails.LdapUserDetailsManager;
 import org.springframework.security.provisioning.UserDetailsManager;
 
+import aero.minova.cas.CustomLogger;
```

Comment:
Dieser Import wird nicht genutzt.

Reply:
<img width="597" alt="image" src="https://github.com/user-attachments/assets/3e5b5796-b197-4f7f-9671-5f4215eedb6d">

Doch wird er ^^

Ist nur dazu gekommen, weil die Klasse in ein anderes package verschoben wurde.